cpp/src/aztec/plonk/proof_system/widgets/random_widgets/permutation_widget_impl.hpp

```diff
@@ -535,116 +533,6 @@ barretenberg::fr ProverPermutationWidget<program_width, idpolys, num_roots_cut_o
     return alpha_base.sqr().sqr();
 }
 
-template <size_t program_width, bool idpolys, const size_t num_roots_cut_out_of_vanishing_polynomial>
-barretenberg::fr ProverPermutationWidget<program_width, idpolys, num_roots_cut_out_of_vanishing_polynomial>::
-    compute_linear_contribution(const fr& alpha, const transcript::StandardTranscript& transcript, polynomial& r)
```

Comment:
This function was only used in the `use_linearization` branch of `compute_linearisation_coefficients`, which has been refactored to `compute_quotient_contribution`.